recipes/julia/meta.yaml

```diff
@@ -0,0 +1,58 @@
+package:
+  name: julia
+  version: "0.5.2"
+
+source:
+  fn: v0.5.2.tar.gz
+  url: https://github.com/JuliaLang/julia/archive/v0.5.2.tar.gz
+  sha256: efa3303d63e39bf3ea06e23ef68807cca0095c7e0507bdf26eae1d5dee7868c7
+  patches:
+  - rpath.patch
+
+build:
+  skip: True  # [py3k or osx or win]
+  rpaths:
+    - lib/julia/
+  number: 0
+
+requirements:
+  build:
+    - gcc  # [linux]
+    - llvm  # [osx]
+    - cmake
+    - zlib 1.2.8
+    - python
+    - autoconf 2.69 pl5.*
+    - automake 1.15 pl5.*
+    - openblas 0.2.19|0.2.19.*
+    - llvmdev 3.9.*
```

Comment:
We upstream everything we can. See all the "Remove for" comments here https://github.com/JuliaLang/julia/blob/4c5cc04156ba074a8baa028c2a8a41b9e70d56ee/deps/llvm.mk#L432-L482 that mean things were successfully upstreamed. The problem is LLVM breaks other things before their next release and we haven't tested or ifdef-ed code sufficiently to work with anything else. We also haven't tested backporting our patches against other LLVM-using projects, they may cause issues in use cases that Julia doesn't care about. Realistically LLVM does not work well as a shared dependency.